gstreamer/src/tags.rs

```diff
@@ -493,14 +279,42 @@ impl TagListRef {
         }
     }
 
-    pub fn get_size<'a, T: Tag<'a>>(&'a self) -> u32 {
+    pub fn get_index_generic<'a>(&'a self, tag_name: &str, idx: u32) -> Option<&'a Value> {
```

Comment:
And also here `SendValue`

Would it be possible to implement the old typed getters/setters around the new generic ones now to get rid of some unsafe code? That would be great